.github/workflows/release.yml

```diff
@@ -238,10 +239,11 @@ jobs:
       - name: Get Tag
         run: echo GIT_TAG="$(node ./.backstage/get_tag.cjs)" >> $GITHUB_ENV
 
-      - uses: actions/download-artifact@v3
+      - uses: actions/download-artifact@v4
         with:
-          name: release
-          path: build-artifacts
+          path: wrappers/node/checksums
+          pattern: release-checksums-*
+          merge-multiple: true
 
```

Comment:
Sorry, another thing here. We were previously downloading `release` to `build-artifacts`, but this is changed to download `release-checksums-*` into `wrappers/node/checksums`. I think this should go back to the former (but as `release-*` unless I'm missing something? (these are the assets uploaded against the github release)

Reply:
Good catch! Looks like that was the result of an unedited copy-and-paste.